Maestro/subscriptions.json

```diff
@@ -649,23 +649,7 @@
         }
       }
     },
-    {
-      "triggerPaths": [
-        "https://github.com/dotnet/extensions/blob/release/2.1//**/*"
-      ],
-      "action": "github-dnceng-branch-merge-pr-generator",
-      "actionArguments": {
-        "vsoSourceBranch": "main",
-        "vsoBuildParameters": {
-          "GithubRepoOwner": "dotnet",
-          "GithubRepoName": "<trigger-repo>",
-          "HeadBranch": "<trigger-branch>",
-          "BaseBranch": "release/3.1",
-          "ExtraSwitches": "-QuietComments"
-        }
-      }
-    },
```

Comment:
This change warrants further conversation. Without actions in dotnet/extensions to handle 'release/3.1' ➡️  'release/2.1` patching / `cherry-pick`ing, what are you proposing to fill the gap left here❔

Reply:
We could add the action to dotnet/extensions, but I'm not actually sure there is a gap. There is no active `main` branch in extensions - anyone creating servicing PRs there will already make the PR into all relevant branches.